cms/templates/settings.html

```diff
@@ -116,9 +116,53 @@ <h3 class="title">${_("Promoting Your Course with {platform_name}").format(platf
           </div>
           % endif
         </section>
-
         <hr class="divide" />
 
+        % if settings.FEATURES.get("ENABLE_CREDIT_ELIGIBILITY", False) and is_credit_course:
+          <section class="group-settings basic">
+            <header>
+              <h2 class="title-2">${_("Credit Eligibility Requirements")}</h2>
+              <span class="tip">${_("Steps needed for credit eligibility")}</span>
+            </header>
+              % if credit_requirements:
+                <ol class="list-input">
+                  % if 'grade' in credit_requirements:
+                    <li class="field text is-not-editable" id="credit-minimum-passing-grade">
+                      <label for="minimum-passing-grade">${_("Minimum Passing Grade")}</label>
+                        % for requirement in credit_requirements['grade']:
+                          <input title="${_('This field is disabled: this information cannot be changed.')}" type="text"
+                              class="long" id="${requirement['name']}" value="${'{0:.0f}%'.format(float(requirement['criteria']['min_grade'])*100)}" readonly />
+                        % endfor
+                    </li>
+                  % endif
+
+                  % if 'proctored_exam' in credit_requirements:
+                    <li class="field text is-not-editable" id="credit-proctoring-requirements">
+                      <label for="proctoring-requirements">${_("Successful Proctored Exam")}</label>
+                        % for requirement in credit_requirements['proctored_exam']:
+                          <input title="${_('This field is disabled: this information cannot be changed.')}" type="text"
+                              class="long" id="${requirement['name']}" value="${requirement['display_name']}" readonly />
+                        % endfor
+                    </li>
+                  % endif
+
+                  % if 'reverification' in credit_requirements:
+                    <li class="field text is-not-editable" id="credit-reverification-requirements">
+                      <label for="reverification-requirements">${_("Successful In Course Reverification")}</label>
+                        % for requirement in credit_requirements['reverification']:
+                          <input title="${_('This field is disabled: this information cannot be changed.')}" type="text"
+                              class="long" id="${requirement['name']}" value="${_("Access to")} ${requirement['display_name']}" readonly />
```

Comment:
translating this might not work if a translation requires the display_name to appear anywhere other than the end of the phrase.  suggested alternative (not tested):
```
value="${_("Access to {display_name}").format(**requirement)}"
```
I also think a translator comment would help to explain what this phrase indicates to the end user.